practice-app/website/templates/base.html

```diff
@@ -43,10 +70,17 @@ <h1>
             <div style="display: table-row">
 
                 <div style="width: 15%; display: table-cell;">
+                    
                     <a href={{ url_for("views.home") }}>Home</a>
-                    <a href={{ url_for('views.login') }}>Login</a>
-                    <a href={{ url_for("views.signup") }}>Signup</a>
+
+
+                    {% if session["token"] %}
```

Comment:
A check for user type can be added here for artists.

Reply:
Requested changes are made to the code in https://github.com/bounswe/bounswe2022group7/pull/179/commits/61b28becf4985b252f74ed1c2c80bcfb270875b3